core/domain/user_services_test.py

```diff
@@ -14,6 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+from contextlib import nested
```

Comment:
so this works -- 
<pre>
with A(), B(), C():
  some_function_call()
</pre>

but I'm unable to find a way to place them in different lines(without using `\`).